### PR TITLE
Updated 'dcos task exec' to print better error message.

### PR DESCRIPTION
**Pull request by klueska**
Running `dcose task exec` against containers launched by the docker
containerizer is currently unsupported. This commits prints a better
error message when attempting to run this command against such a
container.

### CODE REVIEW

File: dcos/mesos.py [hunk 1]
```diff
@@ -1015,8 +1015,16 @@ def __init__(self, task_id, cmd=None, args=None,
         client = DCOSClient()
         master = get_master(client)
 
-        # Get the URL to the agent running the task.
+        # Get the task and make sure its a container launched by the UCR.
         task_obj = master.task(task_id)
+        if "container" in task_obj.dict():
+            if "type" in task_obj.dict()["container"]:
+                if task_obj.dict()["container"]["type"] != "MESOS":
```
Comment (tamarrow) [inline dcos/mesos.py]: lets just use check `if task_obj.dict()["container"]["type"] != "MESOS":` and catch `ValueError`. It's a little cleaner than the nesting `ifs`

Comment (klueska) [inline dcos/mesos.py]: I had something similar to this at first, but the problem is I need to allow everything through except the one case where `task_obj.dict()["container"]["type"] != "MESOS":`. Meaning, I want to allow it to continue even `if "container" not in task_obj.dict()` and `if "type" in task_obj.dict()["container"]`.

I didn't see a better way of getting these semantics.

Comment (tamarrow) [inline dcos/mesos.py]: 👍 lets add a comment?

Comment (klueska) [inline dcos/mesos.py]: sounds good.

**Comment by klueska** [comment]
bumped.

**Pull request merged**